api/types/preflight.go

```diff
@@ -6,30 +6,38 @@ type PostInstallRunHostPreflightsRequest struct {
 
 // HostPreflights represents the host preflight checks state
 type HostPreflights struct {
-	Titles                    []string              `json:"titles"`
-	Output                    *HostPreflightsOutput `json:"output"`
-	Status                    Status                `json:"status"`
-	AllowIgnoreHostPreflights bool                  `json:"allowIgnoreHostPreflights"`
+	Titles                    []string          `json:"titles"`
+	Output                    *PreflightsOutput `json:"output"`
+	Status                    Status            `json:"status"`
+	AllowIgnoreHostPreflights bool              `json:"allowIgnoreHostPreflights"`
```

Comment:
Am I reading correctly that this is the only property that distinguishes `HostPreflights` from `AppPreflights`? I wonder if we really need this here or if there's an alternative method, feels a bit off compared to the rest of the data.